pulumi-orca/src/main/kotlin/com/pulumi/plugin/preconfigured/PluginConfig.kt

```diff
@@ -2,7 +2,7 @@ package com.pulumi.plugin.preconfigured
 
 import com.netflix.spinnaker.kork.plugins.api.PluginConfiguration
 
-@PluginConfiguration("pulumi.PreConfiguredJobStage")
+@PluginConfiguration
```

Comment:
@danielpeach with the new `PluginConfiguration` annotation I had issues running it on Spinnaker 1.20.7. I kept getting a plugin loading error and then found this: https://kb.armory.io/s/article/Exception-starting-plugin-in-Orca. It seems that the solution is to revert back to the deprecated annotation `ExtensionConfiguration`. What's the recommended approach here?

Reply:
I don't think this annotation will be deleted anytime soon, but probably best to use it sooner rather than later. If you have to support 1.20.7 then I would cut a release branch to support it. 1.20.7 is 4 minor versions back (latest is 1.24.2) if that helps.